src/window/index.ts

```diff
@@ -208,6 +210,18 @@ export async function openFirstRunDialog() {
   }));
 }
 
+/**
+ * Open a dialog warning the user that RD cannot be run as root/administrator,
+ * and return once the user has acknowleged this.
+ */
+export async function openDenyRootDialog() {
+  const window = openDialog('DenyRoot', { frame: true });
```

Comment:
Consider providing `width` & `height` to reduce the amount of visible resizing of the window as we fit to content. Width & height can be defined either while creating the window or in the `DenyRoot` component and we should get the same end result.

Reply:
It's weird, it doesn't seem like the `width` property does anything (when passing it to `openDialog`). But the `height` one does... yet it doesn't seem it is needed as much.

Reply:
What does it look like to you regarding resizing? I haven't noticed anything on my Windows or openSUSE installs

Reply:
This is more of an issue on macOS where the window resize is animated. The transition ends up looking quite odd when the window grows from 0 width/height to the content size.

Reply:
https://user-images.githubusercontent.com/835961/195465175-74281633-8dbd-42e6-94c0-69b70d3eb9ea.mp4